### DISCUSSION TITLE
<title>hacked for6mths by github affiliate

### DISCUSSION THREAD

**Discussion by Mrhart8887190** [community/community#45952] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

#I need help. I have been hacked by someone or rather a group that is affiliated with Github. I have been trying to get help I have had 12 phones a computer and 2 tablets that belong to my toddler whom is disabled. I just need help@? Please

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Mrhart8887190 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
